core/core.php

```diff
@@ -15,18 +15,22 @@
  */
 class core
 {
+	private $config;
 	private $db;
 	private $table_prefix;
 	private $path_helper;
 	private $ext_manager;
+	private $avatar_manager;
 
 	public function __construct()
 	{
 		global $phpbb_container;
+		$this->config = $phpbb_container->get('config');
 		$this->db = $phpbb_container->get('dbal.conn');
```

Comment:
Btw when fetching from the default phpbb_container we don't need to pass the service as an argument in the configuration file. So https://github.com/OpenRA/openrauseraccounts/blob/master/config/services.yml#L8-L11 is probably bogus (while https://github.com/OpenRA/openrauseraccounts/blob/master/config/services.yml#L3-L7 is not because in https://github.com/OpenRA/openrauseraccounts/blob/master/controller/main.php#L31 the arguments are actually passed to the constructor). Just in case you didn't know and want to continue working on the codebase. AFAIK phpbb3.2 uses symfony 2.8 so https://symfony.com/doc/2.8/service_container.html is probably right.